lib/stat_tracker.rb

```diff
@@ -3,16 +3,15 @@
 class StatTracker
   attr_reader :games, :teams, :game_teams
   def initialize
-    @games = nil
-    @teams = nil
-    @game_teams = nil
   end
 
   def self.from_csv(csv_hash)
-    @games = CSV.open csv_hash[:games], headers: true, header_converters: :symbol
-    @teams = CSV.open csv_hash[:teams], headers: true, header_converters: :symbol
-    @game_teams = CSV.open csv_hash[:game_teams], headers: true, header_converters: :symbol
+    # @games_input = CSV.open csv_hash[:games], headers: true, header_converters: :symbol
+    @games_input = CSV.foreach(csv_hash[:games], headers: true, header_converters: :symbol).take(200)
```

Comment:
good catch using .foreach rather than .open. Also good idea to use .take() to use in place of creating separate fixture files